### ISSUE TITLE
Request: Secondary Header Menu

### ISSUE THREAD

**Issue by yofiesetiawan** [JoomShaper/helix-ultimate#134]
I would love to request a secondary (when scroll down) header menu with different style (text color, bg color), different logo, different type (image/text), different height...

**Comment by n00bster** [comment]
+1
good suggestions

**Comment by Pawel-Support** [comment]
+1 
I agree, it would be helpful.

**Comment by stevel4857** [comment]
I also agree. I suggested this just now without knowing it was already requested. My old website had a transparent menu and on scroll down the menu became 80% transparent. So, we need not just color options but levels of transparency also. I assume it couldlook like hex but in the background convert to RGB.
